### PR TITLE
Added zk docs

### PR DESCRIPTION
**Pull request by applejag**
Tried out using zk on Emanote, but took some tinkering to get it working.

Though'd I'd share my findings

### CODE REVIEW

File: docs/resources/zk.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+page:
+  headHtml: |
+    <snippet var="js.highlightjs" />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.5.1/languages/ini.min.js"></script>
```
Comment (srid) [inline docs/resources/zk.md:5]: Why not use PrismJS here?

File: docs/resources/zk.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+generic in the sense that it does not lock in to only supporting a certain note
+taking tool, which makes it useful for [Obsidian](https://obsidian.md/),
+[Neuron](https://neuron.zettel.page/), and Emanote notes collections.
+
+## Configuring repo
+
+```toml
+[root]
+foo = "bar"
+```
```
Comment (srid) [inline docs/resources/zk.md]: This TOML config is placed here without any context. What is the reader supposed to do with this? Why does it say "foo"? What is the meaning of this config?

File: docs/resources/zk.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+## Editor integration
+
+`zk` provides a LSP (Language Server Protocol) server to allow auto-completion
+when writing `[[wiki links]]` and `#tags`.
+
+Available plugins:
+
+- [`zk-nvim`](https://github.com/mickael-menu/zk-nvim) for Neovim 0.5+
+- [`zk-vscode`](https://github.com/mickael-menu/zk-vscode) for Visual Studio Code
```
Comment (srid) [inline docs/resources/zk.md]: Link "Visual Studio Code" to the existing VScode page? Likewise, link "Neovim" to `[[vim|Neovim]]`.

**Review by srid — CHANGES_REQUESTED** [review]
Very interesting; I didn't know about zk-vscode ... I should give it a try!

Maybe we can even add `zk` to emanote-template's flake.nix.

File: docs/resources/zk.md (hunk 2 above)
Comment (applejag) [reply docs/resources/zk.md]: Lol I accidentally committed my test code from #320. Nice catch, will remove!

File: docs/resources/zk.md (hunk 3 above)
Comment (applejag) [reply docs/resources/zk.md]: Sure!

File: docs/resources/zk.md (hunk 2 above)
Comment (applejag) [reply docs/resources/zk.md]: Removed

**Comment by srid** [comment]
It would be good to do https://github.com/srid/emanote-template/issues/8 as part of this. I'll try to get to it, unless someone beats me.

**Comment by applejag** [comment]
> Maybe we can even add `zk` to emanote-template's flake.nix.

> It would be good to do [srid/emanote-template#8](https://github.com/srid/emanote-template/issues/8) as part of this. I'll try to get to it, unless someone beats me.

I'll please skip that part. Nix stuff is one of the things I'm still too scared of to try. So don't wait for me to create PR for that :)

File: docs/resources/zk.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
+---
+page:
+  headHtml: |
+    <snippet var="js.highlightjs" />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.5.1/languages/ini.min.js"></script>
```
Comment (srid) [inline docs/resources/zk.md:5]: Imagine if we could just `<snippet var="js.highlightjs-lang.ini" />`. 

(Don't have to do this in this PR.)

Comment (applejag) [reply docs/resources/zk.md:5]: Even better if the snippets could accept arbitrary variables, like

```html
<snippet var="js.highlightjs-lang" lang="ini" />

<!-- uses: -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.5.1/languages/{{ lang }}.min.js"></script>

<!-- results in: -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.5.1/languages/ini.min.js"></script>
```

**Pull request merged**